lily/grob.cc

```diff
@@ -147,6 +147,7 @@ Grob::get_print_stencil () const
       /* A grob has to be visible, otherwise the whiteout property has no effect. */
       /* Calls the scheme procedure stencil-whiteout in scm/stencils.scm */
       if (!transparent && (scm_is_number (get_property ("whiteout"))
+                           || scm_is_pair (get_property ("whiteout"))  /* We actually would need number-pair */
```

Comment:
Shouldn't it be easily possible to create a "number-pair" within that OR expression?
I wouldn't mind one or two additional lines when it helps avoid errors down the road or at least clarifies the code